### PR TITLE
Review branch

### PR DESCRIPTION
**Pull request by solomonetchu**
Adding Youtube page content

### CODE REVIEW

File: sass/_base.scss [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+*::before,
+*::after {
+  margin: 0;
+  padding: 0;
+  box-sizing: inherit;
+}
+
+html {
+  box-sizing: border-box;
+  font-size: 62.5%; // 10px/16px = 62.5% -> 1rem = 10px
```
Comment (solomonetchu) [inline sass/_base.scss:27]: Please do not leave unnecessary comment in your code, it makes it look unprofessional

File: sass/main.scss [hunk 2]
```diff
@@ -0,0 +1,5 @@
+@import "base";
+@import "typography";
+@import "header";
+@import "features";
```
Comment (solomonetchu) [inline sass/main.scss:4]: Please review this sass "import" if you are not using it kindly removed it to had a great presentation.

File: sass/main.scss [hunk 3]
```diff
@@ -0,0 +1,5 @@
+@import "base";
+@import "typography";
```
Comment (solomonetchu) [inline sass/main.scss:2]: Please review this sass "import" if you are not using it kindly removed it to had a great presentation.

**Review by solomonetchu — COMMENTED** [review]
Generally you code looks great. but if you work on the responsiveness it will be better.

Keep up the good work

**Review by leonardodiasb — APPROVED** [review]
# Status: ✔️ Project Approved ✔️ 

Hi @solomonetchu,

Everything is looking great! Please, merge this branch :shipit:
Congratulations! 🎉

## Highlights

 - Polite comments. ✔️

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

 - [OPTIONAL] Try using a couple of emojis and/or gifs in the review. They make the code review more human-friendly 👽🤖.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think the code review was unfair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**